app/components/dem-info.jsx

```diff
@@ -0,0 +1,54 @@
+'use strict'
+import React from 'react';
+import cx from 'classnames';
+import insertSheet from 'react-jss';
+
+
+function DemInfo(props) {
+    const { user, classes, styles, className, ...otherProps } = props;
```

Comment:
since styles is not users (and shouldn't have the s at the end) please remove it.